R4_UK_BRAM_Update_Slot_include

JagatNayak opened the pull request:
Description
----
R4_UK_BRAM_Update_Slot_include

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [x] Screenshot(s) of changes attached after changes merged and published.

JagatNayak commented:
<img width="1143" alt="image" src="https://github.com/cerner/fhir.cerner.com/assets/65965700/d4c02c9b-67b3-43a8-93d3-fe95526dc934">

JagatNayak commented:
Implementation Note: 
<img width="1143" alt="image" src="https://github.com/cerner/fhir.cerner.com/assets/65965700/e2fff0eb-5bf1-4e1b-a6ec-4bcf8f7b58a0">

JagatNayak commented:
Example with Include  when **_include=Slot:schedule** :
<img width="1143" alt="image" src="https://github.com/cerner/fhir.cerner.com/assets/65965700/3be71562-79da-4bd1-95ad-4d66351ddf37">

JagatNayak commented:
Example with Include when _include=Schedule :actor:Practitioner  :
<img width="1143" alt="image" src="https://github.com/cerner/fhir.cerner.com/assets/65965700/14df3557-bee7-4bc4-9d38-54c38f1fdc40">

JagatNayak commented:
Patient Authorization Request when _include=Slot:schedule : 

<img width="1143" alt="image" src="https://github.com/cerner/fhir.cerner.com/assets/65965700/ea642090-e105-4b24-ab11-056f1d147c6e">

JagatNayak commented:
Patient Authorization Request when _include=Schedule:acto:Practitioner :
<img width="1143" alt="image" src="https://github.com/cerner/fhir.cerner.com/assets/65965700/ea25a8a6-4cf7-4054-a47d-083903b42d10">

ashu-bdvt commented on content/millennium/r4/base/workflow/slot.md:
```diff
@@ -91,11 +91,13 @@ Notes:
-* When `_include` is provided in a request to the closed endpoint, the OAuth2 token must include one of `user/Schedule.read`, `patient/Schedule.read`, or `system/Schedule.read` scopes in addition to the Slot scope.
+* When `_include=Slot:schedule` is provided in a request to the closed endpoint, the OAuth2 token must include one of `user/Schedule.read`, `patient/Schedule.read` or `system/Schedule.read` scopes in addition to the Slot scope.
+
+*  When `_include=Schedule:actor:Practitioner` is provided in a request to the closed endpoint, the OAuth2 token must include  one of `user/Practitioner.read`, `patient/Practitioner.read` or `system/Practitioner.read` scopes in addition to the Slot and Schedule scope.
```
```suggestion
*  When `_include=Schedule:actor:Practitioner` is provided in a request to the closed endpoint, the OAuth2 token must include  one of `user/Practitioner.read`, `patient/Practitioner.read` or `system/Practitioner.read` scopes and one of `user/Schedule.read`, `patient/Schedule.read` or `system/Schedule.read` scopes in addition to the Slot scope.
```

JagatNayak commented:
Addressed comment :
<img width="1143" alt="image" src="https://github.com/cerner/fhir.cerner.com/assets/65965700/dd2b4d98-0ecf-4018-9244-bb539ab88811">

JagatNayak commented:
Before Changes :
<img width="783" alt="image" src="https://github.com/cerner/fhir.cerner.com/assets/65965700/0e3475ae-b805-4f1f-85ec-256f18e2f342">

JagatNayak commented:
Before Changes Implementation :
<img width="1159" alt="image" src="https://github.com/cerner/fhir.cerner.com/assets/65965700/59c1dcaf-0ff1-48fe-af07-2bf3dde3b5f9">

Surya-Rnsit commented:
After Release:
<img width="1792" alt="Screenshot 2023-11-10 at 3 25 59 PM" src="https://github.com/cerner/fhir.cerner.com/assets/87635660/1d102bc2-7fcc-4324-94fb-71e25c7e91b0">
<img width="1792" alt="Screenshot 2023-11-10 at 3 24 52 PM" src="https://github.com/cerner/fhir.cerner.com/assets/87635660/3899e404-d05c-4df2-8940-00a22ffeaefb">
